lua/entities/sent_tardis/shared.lua

```diff
@@ -4,15 +4,15 @@ if WireLib then
 else
 	ENT.Base			= "base_gmodentity"
 end 
-ENT.PrintName		= "TARDIS"
+ENT.PrintName		= "Legacy TARDIS"
 ENT.Author			= "Dr. Matt"
 ENT.Contact			= "mattjeanes23@gmail.com"
 ENT.Instructions	= "Use with the sonic or press E to pilot."
 ENT.Purpose			= "Time and Relative Dimension in Space"
 ENT.Spawnable		= true
 ENT.AdminSpawnable	= true
 ENT.RenderGroup = RENDERGROUP_BOTH
-ENT.Category		= "Doctor Who"
+ENT.Category		= "Doctor Who - TARDIS (Legacy)"
```

Comment:
This should not be changed
```suggestion
ENT.Category		= "Doctor Who"
```